modules/n1ql/pages/n1ql-language-reference/with-recursive.adoc

```diff
@@ -235,7 +367,7 @@ WITH RECURSIVE similar_items AS (
   SELECT p1.*, similar_items.lvl+1 as lvl FROM products p1,
     similar_items WHERE p1.id IN similar_items.related_items
 )
-OPTIONS {"levels:2"}
+OPTIONS {"levels":2}
 SELECT * FROM similar_items;
 ---- 
```

Comment:
Show the expected results.

Also, add a line _after_ the query with the CYCLE clause, stating the results are the same as the previous query. You don't need to show the results twice.

```suggestion
----

.Results
[source,json]
----
[
  {
    "similar_items": {
      "category": "Sports",
      "description": "A sturdy and reliable bicycle.",
      "id": 1,
      "lvl": 0,
      "name": "Bicycle",
      "price": 299.99,
      "related_items": [
        2,
        4
      ]
    }
  },
  {
    "similar_items": {
      "category": "Sports",
      "description": "A safety helmet for cycling.",
      "id": 2,
      "lvl": 1,
      "name": "Helmet",
      "price": 49.99,
      "related_items": [
        1,
        3
      ]
    }
  },
  {
    "similar_items": {
      "category": "Sports",
      "description": "Comfortable gloves for cycling.",
      "id": 4,
      "lvl": 1,
      "name": "Gloves",
      "price": 19.99,
      "related_items": [
        1,
        3
      ]
    }
  },
  {
    "similar_items": {
      "category": "Accessories",
      "description": "A convenient water bottle for cycling.",
      "id": 3,
      "lvl": 2,
      "name": "Water Bottle",
      "price": 9.99,
      "related_items": [
        2,
        4
      ]
    }
  },
  {
    "similar_items": {
      "category": "Sports",
      "description": "A sturdy and reliable bicycle.",
      "id": 1,
      "lvl": 2,
      "name": "Bicycle",
      "price": 299.99,
      "related_items": [
        2,
        4
      ]
    }
  }
]
----
```

Reply:
Another thing I've just noticed. In the reporting hierarchy example, I think the `reportsTo` alias is misleading. The query doesn't show who each employee reports to, it shows each employee's reports, i.e. the people they manage. Please can you change `reportsTo` to `reports` in this query, and in the results?

(Also, let's change some of the employee names to add some more diversity!)